gdc_fossa.yaml

```diff
@@ -0,0 +1,2 @@
+---
+scan_image: "harbor.intgdc.com/tool/gdc-fossa-cli-frontend-node:node-12.16.1-yarn-1.22.4"
```

Comment:
Just curious if the node/yarn version should be in sync with the used image for building. If yes, have you considered some possibility (may not even be feasible) to link image version from https://github.com/gooddata/gooddata-js/blob/master/docker/.config#L1 or from some common ancestor space?